staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1879,6 +1879,7 @@ type KubeVirtConfiguration struct {
 	PermittedHostDevices        *PermittedHostDevices `json:"permittedHostDevices,omitempty"`
 	MinCPUModel                 string                `json:"minCPUModel,omitempty"`
 	ObsoleteCPUModels           map[string]bool       `json:"obsoleteCPUModels,omitempty"`
+	MaxDevices                  *int                  `json:"maxDevices,omitempty"`
 }
```

Comment:
I realize "max devices" is the abstraction used internally by virt-handler, but it isn't the best terminology in the context
of a user-facing and cluster-wide terminology.

How about using "MaxVMsPerNode"? I think this better reflects the user's intent here, and also makes it clear that this is a per-node limit.

Reply:
How about `virtualMachineInstancesPerNode`?